config/packages/test/security.yaml

```diff
@@ -1,7 +1,3 @@
 security:
     password_hashers:
-        sha512: sha512
-        Sylius\Component\User\Model\UserInterface: sha512
-
-sylius_user:
-    encoder: sha512
+        Sylius\Component\User\Model\UserInterface: argon2i
```

Comment:
this impacts performance a lot. why not using plainPassword for example.

Reply:
Let's try to switch it in a separate PR, wdyt? It would be easier to track these performance changes (btw, it would be great if we finally have some performance tests in the pipeline 😄)